### PR TITLE
[ES] Add `Gameplay/Dash snapping`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Dash_snapping/es.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Dash snapping
+
+En [osu!catch](/wiki/Game_mode/osu!catch) cada [dash](/wiki/Gameplay/Dash) e [hiperdash](/wiki/Gameplay/Hyperdash) se pueden clasificar en dos snappings diferentes, snap básico o snap avanzado.
+
+## Snap básico
+
+El término **basic-snapped**, o **snap básico**, se usa para cualquier [dash](/wiki/Gameplay/Dash) o [hiperdash](/wiki/Gameplay/Hyperdash) cuyo tiempo entre [objetos activos](/wiki/Beatmapping/Active_object) es al menos dos veces mayor que el tiempo necesario para permitir dashes o hiperdashes, respectivamente. Por ejemplo, un hiperdash entre objetos separados por 250 ms en un Platter se clasifica como un hiperdash básico.
```
Comment (agatemosu) [inline wiki/Gameplay/Dash_snapping/es.md:7]:
```suggestion
El término **snap básico** (*basic-snapped*) se usa para cualquier [dash](/wiki/Gameplay/Dash) o [hiperdash](/wiki/Gameplay/Hyperdash) cuyo tiempo entre [objetos activos](/wiki/Beatmapping/Active_object) es al menos dos veces mayor que el tiempo necesario para permitir dashes o hiperdashes, respectivamente. Por ejemplo, un hiperdash entre objetos separados por 250 ms en un Platter se clasifica como un hiperdash básico.
```

File: wiki/Gameplay/Dash_snapping/es.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Snap avanzado
+
+El término **higher-snapped**, o **snap avanzado**, se usa para cualquier [dash](/wiki/Gameplay/Dash) o [hiperdash](/wiki/Gameplay/Hyperdash) que no cumple con el requisito de ser un snap básico (es decir, el tiempo entre los objetos es más corto que el mínimo para ser clasificado como básico).
```
Comment (agatemosu) [inline wiki/Gameplay/Dash_snapping/es.md:11]:
```suggestion
El término **snap avanzado** (*higher-snapped*) se usa para cualquier [dash](/wiki/Gameplay/Dash) o [hiperdash](/wiki/Gameplay/Hyperdash) que no cumple con el requisito de ser un snap básico (es decir, el tiempo entre los objetos es más corto que el mínimo para ser clasificado como básico).
```

File: wiki/Gameplay/Dash_snapping/es.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+## Tabla de referencia de los snaps
+
+| Dificultad | Dash con un snap básico | Dash con un snap avanzado | Hiperdash con un snap básico | Hiperdash con un snap avanzado |
+| :-- | :-- | :-- | :-- | :-- |
+| **Cup** | - | - | - | - |
+| **Salad** | 250 ms o más | 125-249 ms | - | - |
+| **Platter** | 125 ms o más | 62-124 ms | 250 ms o más | 125-249 ms |
+| **Rain** | 125 ms o más | 62-124 ms | 125 ms o más | 62-124 ms |
```
Comment (agatemosu) [inline wiki/Gameplay/Dash_snapping/es.md:20]:
```suggestion
| **Salad** | 250 ms o más | 125-249 ms | - | - |
| **Platter** | 125 ms o más | 62-124 ms | 250 ms o más | 125-249 ms |
| **Rain** | 125 ms o más | 62-124 ms | 125 ms o más | 62-124 ms |
```

File: wiki/Gameplay/Dash_snapping/es.md (hunk 2 above)
Comment (UnknownOsu) [reply wiki/Gameplay/Dash_snapping/es.md]: Estaba como `snap alto` en los criterios de osu!catch, aunque no me disgustaba `snap avanzado`

File: wiki/Gameplay/Dash_snapping/es.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+En [osu!catch](/wiki/Game_mode/osu!catch), cada [dash](/wiki/Gameplay/Dash) e [hiperdash](/wiki/Gameplay/Hyperdash) se pueden clasificar en dos snappings diferentes, snap básico o snap alto.
+
+## Snap básico
+
+El término **snap básico** se usa para cualquier [dash](/wiki/Gameplay/Dash) o [hiperdash](/wiki/Gameplay/Hyperdash) cuyo tiempo entre [objetos activos](/wiki/Beatmapping/Active_object) es al menos dos veces mayor que el tiempo necesario para permitir dashes o hiperdashes, respectivamente. Por ejemplo, un hiperdash entre objetos separados por 250 ms en un Platter se clasifica como un hiperdash básico.
+
+## Snap alto
+
+El término **snap alto** se usa para cualquier [dash](/wiki/Gameplay/Dash) o [hiperdash](/wiki/Gameplay/Hyperdash) que no cumple con el requisito de ser un snap básico (es decir, el tiempo entre los objetos es más corto que el mínimo para ser clasificado como básico).
```
Comment (agatemosu) [inline wiki/Gameplay/Dash_snapping/es.md:11]:
```suggestion
El término **snap alto** (*higher-snapped*) se usa para cualquier [dash](/wiki/Gameplay/Dash) o [hiperdash](/wiki/Gameplay/Hyperdash) que no cumple con el requisito de ser un snap básico (es decir, el tiempo entre los objetos es más corto que el mínimo para ser clasificado como básico).
```

File: wiki/Gameplay/Dash_snapping/es.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# Dash snapping
+
+En [osu!catch](/wiki/Game_mode/osu!catch), cada [dash](/wiki/Gameplay/Dash) e [hiperdash](/wiki/Gameplay/Hyperdash) se pueden clasificar en dos snappings diferentes, snap básico o snap alto.
+
+## Snap básico
+
+El término **snap básico** se usa para cualquier [dash](/wiki/Gameplay/Dash) o [hiperdash](/wiki/Gameplay/Hyperdash) cuyo tiempo entre [objetos activos](/wiki/Beatmapping/Active_object) es al menos dos veces mayor que el tiempo necesario para permitir dashes o hiperdashes, respectivamente. Por ejemplo, un hiperdash entre objetos separados por 250 ms en un Platter se clasifica como un hiperdash básico.
```
Comment (agatemosu) [inline wiki/Gameplay/Dash_snapping/es.md:7]:
```suggestion
El término **snap básico** (*basic-snapped*) se usa para cualquier [dash](/wiki/Gameplay/Dash) o [hiperdash](/wiki/Gameplay/Hyperdash) cuyo tiempo entre [objetos activos](/wiki/Beatmapping/Active_object) es al menos dos veces mayor que el tiempo necesario para permitir dashes o hiperdashes, respectivamente. Por ejemplo, un hiperdash entre objetos separados por 250 ms en un Platter se clasifica como un hiperdash básico.
```

**Review by agatemosu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**